packages/web/src/scss/components/Collapse/index.html

```diff
@@ -167,6 +167,42 @@ <h2 class="docs-Heading">Collapse with hide trigger when expanded </h2>
 
 </section>
 
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default with helper class for changing content inside toggle</h2>
+
+  <p>
+    Ante rhoncus, tristique nunc et elit id in ex sem consectetur id, consectetur et elit nibh sed ac. Sed ac convallis, sem lobortis convallis et nisi sapien in ex sem ullamcorper
+    iaculis, aliquet sem lobortis pellentesque sit amet. Congue nulla congue, porttitor quis orci quam elit rutrum scelerisque nibh maximus, felis a integer maximus praesent.
+    Metus varius, at sem dui vivamus elementum luctus nisl sit amet dolor sit amet, mauris et iaculis nec fermentum. Augue sapien, nisl vel purus nullam odio morbi a fusce vitae
+    mattis vitae, posuere tristique bibendum aliquam. In donec dolor ut, imperdiet quam fermentum molestie vulputate scelerisque ac nec, tortor mi orci sollicitudin elementum.
+  </p>
+  <div
+    id="CollapseExampleV"
+    class="Collapse"
+  >
+    <div class="Collapse__content">
+      Aliquam varius, consequat posuere a lacinia mauris eu tellus condimentum ut id ante, accumsan vehicula nulla neque. Mauris mi orci, in donec nullam odio leo sapien et vehicula nunc a
+      lacinia, fermentum arcu ullamcorper posuere. Mauris euismod, ac nec ante fermentum praesent nisi commodo neque placerat, vivamus dui et tempus pulvinar suspendisse. Porttitor eget,
+      sollicitudin hendrerit bibendum nulla aliquam sit amet leo vitae, eget consectetur diam a vestibulum. Adipiscing lorem ipsum, arcu condimentum posuere semper morbi condimentum dui,
+      bibendum nunc aenean facilisis. Phasellus euismod, donec sem odio ligula praesent finibus nibh convallis, tristique aliquam sed id tortor sem lobortis.
+    </div>
+  </div>
+  <div class="mb-400">
+    <button
+      type="button"
+      role="button"
+      data-toggle="collapse"
+      data-target="CollapseExampleV"
+      class="Button Button--primary Button--medium"
+    >
+      <span class="accessibility-expanded">Show less</span>
+      <span class="accessibility-collapsed">Show more</span>
```

Comment:
@crishpeen @dlouhak @vmilersky This is what we managed to come up with, having it discussed in person. I still can see at least two other possible ways:

1. be more explicit:
   1. `accessibility-expanded-visible` + `accessibility-collapsed-visible`
   2. `accessibility-expanded-visible` + `accessibility-expanded-hidden`
 2. make it part of `Collapse`: `.Collapse[aria-expanded="true"] .Collapse__expandedText` + `.Collapse[aria-expanded="true"] .Collapse__collapsedText`

The helper class approach (1) makes it usable with other expanding components like `Dropdown`.

We are (or at least I am 🙂) open to discuss better ideas!

Reply:
@adamkudrna I thought the name itself conveyed the status. So for open it is `accessibility-expanded` and for closed it is `accessibility-collapsed`. 
I'm not quite sure about the class names 1.ii. therefore I would rather lean towards 1.i.

I will add `-visible` suffix, ok?

Reply:
What about being more general and using `opened` / `closed` naming? I think that these utilities will be used somewhere else in the future and `open`/`close` status is also more readable. And as a side note we recently renamed all `expanded`/`collapsed` statuses to `isOpen`/`isClosed` so we should be consistent in naming things.

Reply:
For simplicity, @literat and I suggested that these classes should be named `accessibility-open` and `accessibility-closed`. @adamkudrna and @crishpeen are you okay with it like this?

Reply:
`accessibility-open` please (because it's an adjective), otherwise OK ✅.